src/Xunit.Frameworks.Autofac/TestFramework/AutofacTheoryTestCaseRunner.cs

```diff
@@ -26,6 +27,11 @@ internal class AutofacTheoryTestCaseRunner : XunitTheoryTestCaseRunner
             _testClassLifetimeScope = testClassLifetimeScope;
         }
 
+        protected override async Task AfterTestCaseStartingAsync()
+        {
+
+        }
+
```

Comment:
@dabide I originally thought that this fixed the issue because my new unit test passed. But my test appears to have only passed because theories stopped getting run at all. I'll submit a new PR if I do get it working again.